RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedProducer.cc

```diff
@@ -328,6 +328,17 @@ ElectronSeedProducer::fillDescriptions(edm::ConfigurationDescriptions& descripti
     psd0.add<double>("hOverEHFMinE",0.8);
     psd0.add<double>("DeltaPhi2B",0.008);
     psd0.add<double>("PhiMin2B",-0.002);
+
+    psd3.add<std::string>("ComponentName",std::string("SeedFromConsecutiveHitsCreator"));
+    psd3.add<std::string>("propagator",std::string("PropagatorWithMaterial"));
+    psd3.add<double>("SeedMomentumForBOFF",5.0);
+    psd3.add<double>("OriginTransverseErrorMultiplier",1.0);
+    psd3.add<double>("MinOneOverPtError",1.0);
+    psd3.add<std::string>("magneticField",std::string(""));
+    psd3.add<std::string>("TTRHBuilder",std::string("WithTrackAngle"));
+    psd3.add<bool>("forceKinematicWithRegionDirection",false);
+    psd0.add<edm::ParameterSetDescription>("SeedCreatorPSet",psd3);
```

Comment:
Mia, what values should be put for the HLT for the parameters of the new SeedCreatorPSet in the ElectronSeedProducer?